phy/mod_ale_forcing.F90

```diff
@@ -54,10 +55,8 @@ subroutine ale_forcing(m, n, mm, nn, k1m, k1n)
     ! Numeric constants for brine absorption profile.
     real(r8), parameter :: cbra1 = 2._r8**(1._r8/3._r8)
     real(r8), parameter :: cbra2 = cbra1*cbra1/12._r8
-    ! real(r8), parameter :: cbra1 = 2._r8**(1._r8/3._r8), &
-    ! real(r8), parameter :: cbra2 = cbra1*cbra1/288._r8
-    real(r8) :: cpi, gaa, pmax, lei1, lei2, lei, q, q3, pmaxi, nlbot, &
-                dsgdt, dsgds
+    real(r8) :: cpi, gaa, pmax, lei1, lei2, lei, q, q_c, q3, q_c3, pmaxi, &
+                nlbot, & dsgdt, dsgds
```

Comment:
```suggestion
                nlbot, dsgdt, dsgds
```
typo needs fixing

Reply:
Thanks, @TomasTorsvik. I already spotted this typo. Intel strangely accepted this, while gfortran build failed. Fix committed.